File: app/dbmodels/NetworkPortAlias.php
<?php

$table = array(
    'fields'        => array(),
    'oldfields'     => array(),
    'renamefields'  => array(),
    'indexes'       => array(),
    'oldindexes'    => array(),
    'relationships' => array(),
    'model'         => 'NetworkPortAlias',
    'menu'          => ''
);
$table['fields'] = array(
    'id'                     => DBModels::type('increments',
                                               array('visible' => false)),
    'network_port_id'        => DBModels::type('integer'),
    'network_port_alias_id'  => DBModels::type('integer'),
);


$table['relationships'] = array(
    'network_port' => array(
        'type'  => 'belongsTo',
        'item'  => 'NetworkPort'),
    'network_port_alias' => array(
        'type'  => 'belongsTo',
        'item'  => 'NetworkPort'),
);

File: fusionglpi/database/networkport_alias.py
from fusionglpi import db


class NetworkportAlias(db.Model):
    id = db.Column(db.Integer, primary_key=True)
    networkport_id = db.Column(db.Integer, db.ForeignKey('networkport.id'))
    networkport_alias_id = db.Column(db.Integer, nullable=False,
                                     server_default="0")
